moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -614,26 +614,17 @@ void PlanningSceneMonitor::newPlanningSceneWorldCallback(const moveit_msgs::Plan
   }
 }
 
-void PlanningSceneMonitor::collisionObjectFailTFCallback(const moveit_msgs::CollisionObjectConstPtr& obj,
-                                                         tf2_ros::filter_failure_reasons::FilterFailureReason reason)
-{
-  // if we just want to remove objects, the frame does not matter
-  if (reason == tf2_ros::filter_failure_reasons::EmptyFrameID && obj->operation == moveit_msgs::CollisionObject::REMOVE)
-    collisionObjectCallback(obj);
-}
-
 void PlanningSceneMonitor::collisionObjectCallback(const moveit_msgs::CollisionObjectConstPtr& obj)
 {
   if (!scene_)
-  {
     return;
-  }
 
   updateFrameTransforms();
   {
     boost::unique_lock<boost::shared_mutex> ulock(scene_update_mutex_);
     last_update_time_ = ros::Time::now();
-    scene_->processCollisionObjectMsg(*obj);
+    if (!scene_->processCollisionObjectMsg(*obj))
```

Comment:
Should this come with a warning?

Reply:
The warnings were issued in `PlanningScene::processCollisionObjectMsg` already.